README.md

```diff
@@ -1 +1,16 @@
-# sal-backend
\ No newline at end of file
+# sal-backend
```

Comment:
have you had chance to setup SAM? can you include server yaml file along with steps???

Reply:
i cannot try sam as i need to install docker for that dont want to install docker on my work laptop , i have tried sls (serverless) for other projects but the framework for sal-backend project needs to be updated for that , so for now im testing remotely on aws